sdk/identity/identity/src/msal/nodeFlows/msalClient.ts

```diff
@@ -835,6 +817,7 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
         loginHint: options?.loginHint,
         errorTemplate: options?.browserCustomizationOptions?.errorMessage,
         successTemplate: options?.browserCustomizationOptions?.successMessage,
+        //prompt: options?.loginHint ? "login" : "select_account",
```

Comment:
Is this intentional?

Reply:
@christothes I am wondering what is the right thing to do? Should we have a prompt parameter or no? Having a prompt parameter means that the login dialog box will always be shown.

Reply:
It should only prompt if no loginHint was supplied.